.travis.yml

```diff
@@ -7,12 +7,12 @@ python:
   - '3.5'
   - '3.6'
   - '3.7'
+  - '3.8'
   - 'pypy3.5'
-  - '3.8-dev'
 
-matrix:
-  allow_failures:
-    - python: '3.8-dev'
+#matrix:
```

Comment:
Why not removing this instead of commenting out?

Reply:
I thought that we will need this again once 3.9-dev is available.

Reply:
That's what git history is for. :P